lookup_plugins/oo_option.py

```diff
@@ -17,7 +17,7 @@
 * if none of the above conditions are met, empty string is returned
 '''
 
-from ansible.utils import template
+from ansible import template
```

Comment:
@detiber I tried this and a couple of other things, but to no avail.  I continue to get the same failure.  I went as far as removing the try statement just in case hoping for a new error but it didn't appear to help me at all.